src/shared/util/configHelper.ts

```diff
@@ -77,42 +77,20 @@ export function setAutoCollectConsole(options: ApplicationInsightsOptions, value
 }
 
 export function enableAutoCollectExternalLoggers(options: ApplicationInsightsOptions, value: boolean) {
-    options.logInstrumentations = {
-        ...options.logInstrumentations,
+    options.logInstrumentationOptions = {
+        ...options.logInstrumentationOptions,
         winston: { enabled: value },
         bunyan: { enabled: value },
     }
 }
 
 export function enableAutoCollectConsole(options: ApplicationInsightsOptions, value: boolean) {
```

Comment:
Doesn't look like this function is necessary anymore as `enableAutoCollectConsole()` runs inside of the config.ts file.

Reply:
As a follow up if have all of these functions used/written in the config.ts file we may be able to eliminate this helper file.